ehitchhiking-ui/src/app/shared/common/styles/colour-constants.scss

```diff
@@ -0,0 +1,147 @@
+$light-blue: #172651;
+$blue: #0a1339;
+$dark-blue: #0a122a;
+
+$green: #698f3f;
+$light-green: #84b149;
+$dark-green: #55773a;
+
+$beige: #e7decd;
+$brown: #804e49;
+$dark-brown: #4b2f24;
+
+$light-red: #FB5B74;
+$red: #F20026;
+$dark-red: #B5001D;
+
+$custom-primary-colour: (
```

Comment:
pls rename to ehitchhiking-primary-color-setting

Reply:
done